main.go

```diff
@@ -395,3 +403,42 @@ func createPullRequestBranch(repo, pr, action string) error {
 	log.Info("Pipeline is expected to start automatically")
 	return nil
 }
+
+func stopStalledPipelines (client *gitlab.Client, vars gitlab.PipelineVariableList) error {
+	integrationPipelinePath := "Northern.tech/Mender/mender-qa"
+
+	/* TODO: check also for gitlab.Pending? */
+	status := gitlab.Running
+	opt := &gitlab.ListProjectPipelinesOptions{
+		Status: &status,
```

Comment:
I would also add `Username: "mender-test-bot"` here. I think it should never cancel a manually triggered job, even if it matches the variables (someone might have started it to compare an earlier SHA, for example).

Reply:
Good precaution measure. Done.